screed/tests/__main__.py

```diff
@@ -1,8 +1,7 @@
-import os
 import sys
 
 if __name__ == '__main__':
-    if len(sys.argv) == 1:
-        sys.argv.append(os.path.dirname(__file__))
-    import nose
-    nose.main()
+    import pytest
+#    errno = pytest.main(['-m', '"not known_failing"', '-v'])
```

Comment:
It would be nice if `python -m screed.tests` did The Right Thing, but not a primary priority IMO. I'd say let's open a ticket on it.